libs/iframe-transport/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/iframe-transport",
-  "version": "1.0.2",
+  "version": "1.2.0",
```

Comment:
We probably forgot to bump the version (it should be bumped by release-please actually).
Because in NPM it's 1.1.0 now.

<img width="782" alt="image" src="https://github.com/user-attachments/assets/ad9ad550-6109-4333-a350-4e9b54e24098" />